draw-lib/racket/draw/private/dc.rkt

```diff
@@ -1360,7 +1360,6 @@
                                  [(make-or-false font%) [use-font font]]
                                  [any? [combine? #f]]
                                  [exact-nonnegative-integer? [offset 0]])
-      (check-ok 'get-text-extent)
```

Comment:
This check needs to stay, right? I'm guessing that you removed it on the way to the actual repair.